app/src/main/java/io/neurolab/activities/DataLoggerActivity.java

```diff
@@ -55,6 +55,8 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     private void showLoggedDataList(File appDir) {
+        filesList.clear();
+        fileList.clear();
```

Comment:
make a seperate method for these something like `clearRedundantFiles()`

Reply:
Done with changes! Please check.